ncsa_storage/TODO

```diff
@@ -0,0 +1 @@
+pv are RWO at ncsa whereas pvc a ReadByMAny, this seems blocking...
```

Comment:
"pvc a ReadByMAny" ...🤔... should be "pvc are ReadByMany"? (a -> are)?

Reply:
TODO was a temporary file, I have removed it. Thanks